### PR TITLE
Reviews the Overview page (sync with NAV, add descriptions)

### PR DESCRIPTION
**Pull request by ignasi35**
Random improvements.

### CODE REVIEW

File: docs-source/docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,19 +1,20 @@
 = Akka Platform Guide
-:toc:
-:toc-title: ON THIS PAGE
-:toclevels: 2
-:description: Examples projects to illustrate features of Akka
+:description: Example projects to illustrate features of Akka
```
Comment (ignasi35) [inline docs-source/docs/modules/ROOT/pages/index.adoc:2]: Having a TOC on the Overview page didn't make much sense.

File: docs-source/docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 
-xref:shopping-microservices:index.adoc[Microservices based on Event sourcing and CQRS example]
+=== xref:shopping-microservices:index.adoc[Microservices based on Event Sourcing and CQRS example]
 
-This example illustrates how to implement Event sourcing with Akka Persistence and CQRS using Akka Projections.
+This example illustrates how to implement Event Sourcing with Akka Persistence and CQRS using Akka Projections.
 
+=== * xref:how-to:index.adoc[]
+
+A collection of HOWTO guides to help you introduce a new feature or architectural improvement on an existing codebase.
```
Comment (ignasi35) [inline docs-source/docs/modules/ROOT/pages/index.adoc:19]: Each of the three sections is now a title and has some small intro.

File: docs-source/docs/modules/ROOT/partials/include.adoc [hunk 3]
```diff
@@ -1,22 +1,9 @@
...
-:cli-plugin:                   kubectl
-
 :latest-openshift-platform-version:        3.11
 :minimum-kubernetes-on-openshift-version:  1.11
 :minimum-openshift-platform-version:       3.11
 :minimum-platform-version-major-dot-minor: 3.11
-:minimum-sbt-version:                      0.13.7
-:latest-sbt-version: 					   1.3.8	 
 
 // link shortcuts for `link:{akka}/typed/index.html` notation
```
Comment (ignasi35) [inline docs-source/docs/modules/ROOT/partials/include.adoc:8]: Remove copy/pasted values we're unlikely to need in the short term.

File: docs-source/docs/modules/shopping-microservices/pages/index.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 include::ROOT:partial$include.adoc[]
 
-This example illustrates how multiple small microservices can interact to implement a system. They use Akka Persistence to enables Event sourcing and Akka Projection to implement CQRS for the event-sourced entities.
+This example illustrates how multiple small microservices can interact to implement a system. They use Akka Persistence to enables Event Sourcing and Akka Projection to implement CQRS for the event-sourced entities.
```
Comment (ignasi35) [inline docs-source/docs/modules/shopping-microservices/pages/index.adoc:8]: Capitalize `S` on `Event Sourcing`.

**Review by ennru — APPROVED** [review]

**Pull request merged**